include/RevCPU.h

```diff
@@ -118,6 +118,7 @@ class RevCPU : public SST::Component{
     {"RDMAPerCycle",    "Number of RDMA messages per cycle to inject",  "1"},
     {"testIters",       "Number of PAN test messages to send",          "255"},
     {"splash",          "Display the splash logo",                      "0"},
+    {"disable_coproc_clock",  "Disables the manual clocking of coprocessor subcomponents", "0"},
```

Comment:
kind of a nitpick but should we keep with camel case for rev options?

Reply:
good call; fixed in 3c3ac59